src/utils/schemas.ts

```diff
@@ -0,0 +1,50 @@
+import { z } from 'zod';
+
+export const registerSchema = z
+    .object({
+        firstName: z
+            .string({
+                required_error: 'First name is required.',
+                invalid_type_error: 'You must enter a valid name.',
+            })
+            .min(2, {
+                message: 'First name must contain at least 2 characters.',
+            })
+            .max(30),
+        lastName: z
+            .string({
+                required_error: 'Last name is required.',
+                invalid_type_error: 'You must enter a valid name',
+            })
+            .min(2, {
+                message: 'Last name must contain at least 2 characters.',
+            })
+            .max(30),
+
+        email: z
+            .string({ required_error: 'Email is required.' })
+            .email('Please enter a valid email.'),
+
+        password: z
```

Comment:
I like your password constraints. It made me feel like, I am registering a NASA application 😊